index.html

```diff
@@ -1448,17 +1456,25 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
   <pre class="idl">
     typedef (DOMString or ArrayBuffer or NFCMessage) NFCMessageSource;
 
+    [Constructor(), SecureContext, Exposed=Window]
+    interface NFCWriter {
+      Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
+    };
+
     [SecureContext]
```

Comment:
The NFC interface should either have `[Exposed=Window]` or `[NoInterfaceObject]` (I guess the latter?). Anyway, the extended attributes for NFCWriter LGTM, and I guess NFC will go away when you add NFCReader.